basic/exercise1/script.js

```diff
@@ -16,12 +16,23 @@ function formatDate(date) {
     var time = `${formatWeekDays} ${formatHours} ${ampm} ${formatMinutes}: ${formatSeconds}`;
     return time;
 };
+function formatshortDate(date) {
+    var month = formatAsTwoDigits(date.getMonth());
+    var day = formatAsTwoDigits(date.getDay());
+    var year = date.getFullYear();
+    var shortDate = `${month}-${day}-${year}`;
+    return shortDate;
 
-function formatAndDisplayDate(date, id) {
-    var fdate = formatDate(date);
-    document.getElementById(id).innerHTML = fdate;
 };
 
-formatAndDisplayDate(new Date(2017, 2, 2, 2, 2, 2), "previousDate");
-formatAndDisplayDate(new Date(), "currentDate");
-formatAndDisplayDate(new Date(2017, 11, 11, 11, 11, 11), "nextDate");
\ No newline at end of file
+function formatAndDisplayDate(date, id, formatFunction) {
+    var fdate = formatFunction(date);
+    document.getElementById(id).innerHTML = fdate;
+};
```

Comment:
Please put new empty line at the end - please separate function with empty line, as it is more readable.